### ISSUE TITLE
Codemod error - Cannot find module '@ag-grid-devtools\codemods\version\31.3.0'

### ISSUE THREAD

**Issue by timabb031** [ag-grid/ag-grid#8331]
**I'm submitting a ...**  
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

I've followed the instructions for upgrading AG Grid from 31.2.0 to 32.0.1 on https://www.ag-grid.com/react-data-grid/upgrading-to-ag-grid-32/, but the code mod tool produces the error highlighted in the screenshot below.

![image](https://github.com/user-attachments/assets/7bac7001-8013-4a51-aa6e-05717e608e42)

* OS is Windows, IDE is VS Code, Package Manager is npm 9.6.7 on Node 20.11.1

* **AG Grid version:** 32.0.1 (React)

* **Language:** TypeScript 5.0.2

**Comment by timabb031** [comment]
It appears v32.0.1 of the codemod runner works, but v32.0.2 (and v32.0) do not

Works:
`npx @ag-grid-devtools/cli@32.0.1 migrate --from=31.2.0`

Doesn't work:
`npx @ag-grid-devtools/cli@32.0 migrate --from=31.2.0`
`npx @ag-grid-devtools/cli@32.0.2 migrate --from=31.2.0`

**Comment by kiril-matev** [comment]
Hello,

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
**AG-12252 When using Codemods on Windows, error is shown - Cannot find module '@ag-grid-devtools\codemods\version**

We have now fixed this issue in the latest release of the codemods - v32.0.3, which is now available:
https://github.com/ag-grid/devtools/releases/tag/32.0.3

Thanks again for bringing this up with us - we appreciate your time!
